app/components/Exchange/MyMarkets.jsx

```diff
@@ -58,12 +59,40 @@ class MarketGroup extends React.Component {
         return (
             !utils.are_equal_shallow(nextState, this.state) ||
             !utils.are_equal_shallow(nextProps.markets, this.props.markets) ||
+            nextState.marketsCache !== this.state.marketsCache ||
```

Comment:
This will always return true, use utils.are_equal_shallow to compare arrays or objects.